flightmate/src/main/java/com/flightmate/servlets/LoginServlet.java

```diff
@@ -12,46 +12,68 @@
 
 import com.flightmate.beans.User;
 import com.flightmate.dao.UserDao;
+import com.flightmate.libs.Role;
 import com.flightmate.libs.Validation;
 import com.flightmate.libs.services.SessionService;
-
 @WebServlet("/login")
 public class LoginServlet extends Validation {
-	private static final long serialVersionUID = 1L;
-	
-	@Override
-	protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws ServletException, IOException {
-		User user = SessionService.srv.getSessionUser(req);
-		
-		if (user != null) {
-			resp.sendRedirect("./dashboard");
-			return;
-		}
-		
-		req.getRequestDispatcher("login.jsp").forward(req, resp);
-	}
+    private static final long serialVersionUID = 1L;
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws ServletException, IOException {
+        User user = SessionService.srv.getSessionUser(req);
+        
+        // If user is already logged in, redirect to dashboard
+        if (user != null) {
+            resp.sendRedirect("./dashboard");
+            return;
+        }
+        
+        // Forward to login page
+        req.getRequestDispatcher("login.jsp").forward(req, resp);
+    }
+
+    @Override
+    protected void doPost(HttpServletRequest req, HttpServletResponse resp) throws ServletException, IOException {
+        // Hardcoded credentials for temporary login
```

Comment:
Is this meant to be temporary? Or be part of the actual code? Mind commenting this out, or removing it if it was only for you?